jenkins/ci.suse.de/cloud-automation-pr-trigger.yaml

```diff
@@ -28,7 +28,7 @@
           ## THIS IS A TRIGGER JOB ONLY - NO WORKER CODE IN HERE - DO NOT DARE TO ADD SOME
           ## TRIGGER JOBS ARE MOVED TO A DEDICATED TRIGGER NODE
           ## THIS JOB IS ABOUT PRs FOR THE AUTOMATION REPO
-          [[ $NODE_NAME != 'cloud-trigger' ]] && exit 99
+          [[ $NODE_NAME =~ cloud.?trigger ]] || exit 99
```

Comment:
... or drop these lines entirely and instead add a test to the make file (that gets run during test and deployment) that verifies all trigger jobs to be bound to the "cloud-trigger" label.